tests/Application/Console/Formatters/FormatResolverTest.php

```diff
@@ -87,9 +87,21 @@ public function testItCreateAGithubActionFormatter(): void
         self::assertInstanceOf(GithubAction::class, $formatters[0]);
     }
 
+    public function testItCreateACodeClimateFormatter(): void
+    {
+        $input = new ArrayInput(['--format' => ['codeclimate']], AnalyseDefinition::get());
+
+        $formatter = FormatResolver::resolve($input, $this->output, $this->consoleOutput);
+
+        self::assertInstanceOf(Multiple::class, $formatter);
+        $formatters = $this->getFormattersInMultiple($formatter);
+        self::assertCount(1, $formatters);
+        self::assertInstanceOf(CodeClimate::class, $formatters[0]);
```

Comment:
Use missing ?

Reply:
Can you expand? This was just a copy and paste of the Json test:

```
    public function testItCreateAJsonFormatter(): void
    {
        $input = new ArrayInput(['--format' => ['json']], AnalyseDefinition::get());

        $formatter = FormatResolver::resolve($input, $this->output, $this->consoleOutput);

        self::assertInstanceOf(Multiple::class, $formatter);
        $formatters = $this->getFormattersInMultiple($formatter);
        self::assertCount(1, $formatters);
        self::assertInstanceOf(Json::class, $formatters[0]);
    }
```

Reply:
The `use NunoMaduro\PhpInsights\Application\Console\Formatters\CodeClimate;` is missing in this file I mean.

Reply:
I just fix it :wink:

Reply:
Oh duh